rpm/build-rpms

```diff
@@ -33,7 +33,7 @@ case ${pkg} in
 	# Tar up the Git source, with the naming that rpmbuild expects.
 	dir=${pkg}-${version}
 	cp -a ${repo} /tmp/$dir
-	tar -C /tmp --exclude-vcs -czf /tmp/rpmbuild/SOURCES/${dir}.tar.gz $dir
+	tar -C /tmp --exclude-vcs --exclude=vendor --exclude=.go-pkg-cache --exclude=report -czf /tmp/rpmbuild/SOURCES/${dir}.tar.gz $dir
```

Comment:
Wondering if we need a counterpart here for more things that are excluded in the Debian case: `-I'bin/calico-felix-*' -I'*.d' -I'*.ll'`

Reply:
Yep, looks like those files end up in the src RPM.